lib/Element/Uri.php

```diff
@@ -59,7 +59,7 @@ public function xmlSerialize(Xml\Writer $writer): void
     {
         $writer->text(
             resolve(
-                $writer->contextUri,
+                $writer->contextUri ?? '',
```

Comment:
`contextUri` starts life set to `null`.
phpstan can't be sure that it is no longer `null`
The safest thing here is to pass the empty string if it is currently `null`